kapitan/targets.py

```diff
@@ -48,16 +48,16 @@ def compile_targets(inventory_path, search_path, output_path, parallel, targets,
     multiprocessing pool with parallel number of processes.
     kwargs are passed to compile_target()
     """
-    target_path = "targets"
     # temp_path will hold compiled items
     temp_path = tempfile.mkdtemp(suffix='.kapitan')
+
+    target_objs = load_target_inventory(inventory_path, targets)
```

Comment:
Initializing `target_objs` before `multiprocessing.Pool` fork is the magic sauce